x-pack/legacy/plugins/apm/public/components/app/ServiceMap/Popover/ServiceMetricList.tsx

```diff
@@ -0,0 +1,179 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  EuiFlexGroup,
+  EuiLoadingSpinner,
+  EuiFlexItem,
+  EuiBadge
+} from '@elastic/eui';
+import lightTheme from '@elastic/eui/dist/eui_theme_light.json';
+import { i18n } from '@kbn/i18n';
+import { isNumber } from 'lodash';
+import React from 'react';
+import styled from 'styled-components';
+import { ServiceNodeMetrics } from '../../../../../server/lib/service_map/get_service_map_service_node_info';
+import {
+  asDuration,
+  asPercent,
+  toMicroseconds,
+  tpmUnit
+} from '../../../../utils/formatters';
+import { useUrlParams } from '../../../../hooks/useUrlParams';
+import { useFetcher } from '../../../../hooks/useFetcher';
+
+function LoadingSpinner() {
+  return (
+    <EuiFlexGroup
+      alignItems="center"
+      justifyContent="spaceAround"
+      style={{ height: 170 }}
+    >
+      <EuiLoadingSpinner size="xl" />
+    </EuiFlexGroup>
+  );
+}
+
+const ItemRow = styled('tr')`
+  line-height: 2;
+`;
+
+const ItemTitle = styled('td')`
+  color: ${lightTheme.textColors.subdued};
+  padding-right: 1rem;
+`;
+
+const ItemDescription = styled('td')`
+  text-align: right;
+`;
+
+const na = i18n.translate('xpack.apm.serviceMap.NotAvailableMetric', {
+  defaultMessage: 'N/A'
+});
+
+interface MetricListProps {
+  serviceName: string;
+}
+
+export function ServiceMetricList({ serviceName }: MetricListProps) {
+  const {
+    urlParams: { start, end, environment }
+  } = useUrlParams();
+
+  // FIXME: Don't fetch if it's not a service. Perhaps this fetching should go
+  // in ServiceMetricList instead.
```

Comment:
seems this fetch is already in `ServiceMetricList`

Reply:
Code comments be like:
![image](https://user-images.githubusercontent.com/9912/72379467-1cf76580-36d9-11ea-9eb4-a9c8282c23a7.png)